module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraScopedRoleMembership.md

```diff
@@ -46,22 +47,31 @@ Connect-Entra -Scopes 'RoleManagement.ReadWrite.Directory'
 $User = Get-EntraUser -SearchString 'MarkWood'
 $Role = Get-EntraDirectoryRole -Filter "DisplayName eq 'User Administrator'"
 $Unit = Get-EntraAdministrativeUnit -Filter "DisplayName eq 'New MSAdmin unit'"
-$RoleMember = New-Object -TypeName Microsoft.Open.MSGraph.Model.MsRolememberinfo.RoleMemberInfo
+$RoleMember = New-Object -TypeName Microsoft.Open.MSGraph.Model.MsRolememberinfo
 $RoleMember.Id = $User.ObjectID
-Add-EntraScopedRoleMembership -Id $Unit.Id -RoleId $Role.ObjectId -RoleMemberInfo $RoleMember
+$params = @{
+    ObjectId = $unit.ObjectId
+    RoleObjectId = $Role.ObjectId
+    RoleMemberInfo = $RoleMember
+}
+Add-EntraScopedRoleMembership @params
 ```
 
 ```Output
-AdministrativeUnitId     RoleId  
---------------------------            ------------  
-11bb11bb-cc22-dd33-ee44-55ff55ff55ff  22cc22cc-dd33-ee44-ff55-66aa66aa66aa
+Id                                                                AdministrativeUnitId                 RoleId
+--                                                                --------------------                 ------
+dddddddddddd-bbbb-aaaa-bbbb-cccccccccccc aaaaaaaa-bbbb-aaaa-bbbb-cccccccccccc bbbbbbbb-1111-2222-3333-cccccccccccc
 ```
 
 The example shows how to add a user to the specified role within the specified administrative unit.
 
+- `-ObjectId` Paramater specifies the ID of an administrative unit.
+- `-RoleObjectId` Paramater specifies the ID of a directory role.
+- `-RoleMemberInfo` Paramater specifies a RoleMemberInfo object.
+
 ## Parameters
 
-### -AdministrativeUnitId
+### -AdministrativeUnitObjectId
```

Comment:
This should be removed. ObjectId is the right param.

Reply:
Changes done